nservicebus/learning-transport/learning-transport-warning.include.md

```diff
@@ -0,0 +1 @@
+DANGER: The [Learning Transport](/nservicebus/learning-transport/) is not for production use. It is designed for short term learning and experimentation purposes. It should also not be used as a longer-term development transport, i.e. the same transport used in production should be used in development and debug scenarios. Do not use this transport to perform any kind of performance analysis.
```

Comment:
How long is longer term? Does the statement

> the same transport used in production should be used in development and debug scenarios.

cover it? Should we say something like

> Select a production transport before developing features.